com.warlordsoftwares.media-downloader.yml

```diff
@@ -0,0 +1,74 @@
+app-id: com.warlordsoftwares.media-downloader
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '5.15-23.08'
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+command: youtube-dl-pro.sh
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-download
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-run/gvfsd
```

Comment:
You are setting `screenshot-directory=~/Pictures/UMD4-MPV-screenshots` as the screenshot directory, so I assume mpv requires fs access for that path?

Reply:
Actually i dont need to change the default path for screenshot! Thanks good point! +1